src/components/messageView/messageView.js

```diff
@@ -11,33 +11,39 @@ import { helpers } from '../../common';
  * Render a message view.
  *
  * @param {object} props
+ * @param {Node} props.children
  * @param {Node|Function} props.icon
  * @param {string} props.message
  * @param {string} props.pageTitle
  * @param {string} props.title
  * @returns {Node}
  */
-const MessageView = ({ icon, message, pageTitle, title }) => (
+const MessageView = ({ children, icon, message, pageTitle, title }) => (
   <PageLayout>
     <PageHeader>{pageTitle || helpers.UI_DISPLAY_NAME}</PageHeader>
-    <EmptyState variant={EmptyStateVariant.full} className="fadein">
-      {icon && <EmptyStateIcon icon={icon} />}
-      {title && (
-        <Title headingLevel="h2" size="lg">
-          {title}
-        </Title>
+    <PageSection>
+      {(children && children) || (
```

Comment:
required: 
1. So little known, and not every repo has this, but we have access to the nullish coalescing operator through our build setup. It should help clean up this syntax. Instead it'd be...
   ```
   { children ?? (
     <EmptyS...
   ```
1. And it would probably be a good idea to drop in a quick unit snapshot test around `children`

Reply:
Oh man what I did with `(children && children) ||` was totally redundant.  Should've just been `children ||`.

Reply:
that totes works too!